ques.txt

```diff
@@ -0,0 +1,14 @@
+hey all, assigingn some questions/ Assigments to u.
+If u work on them , helps in achieving better results!
+
+Implement the following JSON problem and use Mock for testing your code.
+
+Write a piece of code that will query a URL that returns JSON and can parse the JSON string to pull out pieces of information. The information that should be parsed and returned is the pageid and the list of “See Also” links. Those links should be formatted to be actual links that can be used by a person to find the appropriate article. 
+
+Use the Wikipedia API for the query. A sample query is: 
+http://en.wikipedia.org/w/api.php?format=json&action=query&titles=SMALL&prop=revisions&rvprop=content 
```

Comment:
U can refer to any other link. handle exceptions as well!!

Reply:
Ok .. will take other things as input